docs/extending/developing-code/extending/case-studies/KroneckerProduct/checking.rst

```diff
@@ -0,0 +1,58 @@
+Checking the ``KroneckerProduct[]`` Skeletal Version
+====================================================
+
+As before with ``Undefined`` although there is code to wrote, we have
+entered enough to be able to see that the builtin has been added, and
+that we can get some information about it.
+
+So again, let us try this much. Fire up a a version Django interface that has
```

Comment:
try this first?

Reply:
A couple of things. I am a believer of do little steps and see progress or catch mistakes early. 

To get this process, what I did, really, was record what I was doing as I was doing it and this was the order that I did things.